index.html

```diff
@@ -6,14 +6,324 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
-    <!-- Add a link to your custom CSS here -->
+    
+    <!-- Importing Google fonts  -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville&display=swap" rel="stylesheet">
+    
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&display=swap" rel="stylesheet">
+    
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300&display=swap" rel="stylesheet">
 
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    
+                        <!-- Adding a nav bar and a toggler -->
+<header>
+    <section class="fixed-top bg-light">
+        <aside class="d-none d-lg-flex justify-content-end mx-5 pt-3">
+            <a class="booking" href="#">Book an Appointment</a>
```

Comment:
I think I would have chosen to use the `button` tag here. Looking at the original website, this certainly looks like it's a button, and there's an expectation that when I click on it. But what you have here works too.

One thing I did notice is that in the last section of the page there is a button that looks exactly like this one but it has a different class (this is called booking and the other is called view-article). 

You could reduce repetition if you have just one class that had the button styling and then you reused it on the different elements. That way you don't repeat any code and if you ever need to update the style of a button on your page, you only need to do it in one place.

Reply:
Hi Louise, I have different class names for each button as each one of them had few different stlyes such as size and fonts so they look a bit different from each other.
Hence, the reason not knowing how to have a same class name for all the buttons . But overlall, I agree with your feedback.